go-controller/pkg/util/kube_test.go

```diff
@@ -61,7 +61,7 @@ func TestNewClientset(t *testing.T) {
 
 	for i, tc := range tests {
 		t.Run(fmt.Sprintf("%d:%s", i, tc.desc), func(t *testing.T) {
-			res, e := NewClientset(&tc.inpConfig)
+			res, _, e := NewClientset(&tc.inpConfig)
```

Comment:
nit: assert the return value is not `nil` after the else condition on L68

Reply:
done